### PR TITLE
Split collection notification into 3 separate notifications

### LINKED ISSUE #882: Make text more specific for Email notifications: "Collection Manager: Anyone creates a draft, publishes, opens a new item"
My tester found it confusing that regardless of what the action taken was, all emails received had this same line instead of being specific as to what had happened:

"The Depositor Amy E. Hodge has created a draft / submitted a deposit / started a new version in the A brand new collection for 2021 collection."

I'd like the email to specify which action was taken. The options should be:

- When a user opens a new item and saves it as a draft, the email should say, "The Depositor [depositor’s preferred name] has created a new draft item in the [collection.name] collection."
- When a user deposits an item, the email should say, "The Depositor [depositor’s preferred name] has deposited an item in the [collection.name] collection."
- When a user opens a new version of an existing item, the email should say "The Depositor [depositor’s preferred name] has started a new version of an item in the [collection.name] collection."

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

Each is specific to a particular situation.
Fixes #882

### CODE REVIEW

File: app/controllers/works_controller.rb [hunk 1]
```diff
@@ -148,7 +148,7 @@ def create_new_version(previous_version)
     previous_version.dup.tap do |work_version|
       work_version.state = 'version_draft'
       work_version.version = previous_version.version + 1
-      CollectionObserver.collection_activity(work_version, nil)
+      CollectionObserver.new_version_draft_created(work_version, nil)
```
Comment (ndushay) [inline app/controllers/works_controller.rb:151]: would "version_draft_created" be sufficient, as "created" implies "new"?   "updated" would imply changes ...

File: app/mailers/collections_mailer.rb [hunk 2]
```diff
@@ -48,7 +48,23 @@ def review_access_removed_email
       'collection in the SDR')
   end
 
-  def collection_activity
+  def new_first_draft_created
```
Comment (ndushay) [inline app/mailers/collections_mailer.rb:51]: would "first draft created" be sufficient, as "created" implies new?   "updated" would imply changes ...

File: app/services/collection_observer.rb [hunk 3]
```diff
@@ -3,12 +3,21 @@
 
 # Actions that happen when something happens to a collection
 class CollectionObserver
-  def self.collection_activity(work_version, _transition)
-    depositor = work_version.work.depositor
-    collection = work_version.work.collection
-    collection.managed_by.reject { |manager| manager == depositor }.each do |user|
-      mailer = CollectionsMailer.with(user: user, collection_version: collection.head, depositor: depositor)
-      mailer.collection_activity.deliver_later
+  def self.new_first_draft_created(work_version, _transition)
```
Comment (ndushay) [inline app/services/collection_observer.rb:6]: see naming comment above.

File: app/services/collection_observer.rb [hunk 4]
```diff
@@ -3,12 +3,21 @@
...
+    end
+  end
+
+  def self.item_deposited(work_version, _transition)
+    collection_managers_excluding_depositor(work_version).each do |user|
+      mailer_with_depositor(user: user, work_version: work_version).item_deposited.deliver_later
+    end
+  end
+
+  def self.new_version_draft_created(work_version, _transition)
```
Comment (ndushay) [inline app/services/collection_observer.rb:18]: see naming comment above.

**Review by ndushay — CHANGES_REQUESTED** [review]
This looks fine to me, but I am finding a couple of method names a little too tortured.  YMMV.

**Comment by jcoyne** [comment]
> This looks fine to me, but I am finding a couple of method names a little too tortured. YMMV.

@ndushay yes, good suggestions. changes made.

**Review by ndushay — APPROVED** [review]

**Pull request merged**